dist/lozad.min.js

```diff
@@ -1,4 +1,9 @@
+!function(t,e){"object"==typeof exports&&"undefined"!=typeof module?module.exports=e():"function"==typeof define&&define.amd?define(e):t.lozad=e()}(this,function(){"use strict";var s=Object.assign||function(t){for(var e=1;e<arguments.length;e++){var r=arguments[e];for(var o in r)Object.prototype.hasOwnProperty.call(r,o)&&(t[o]=r[o])}return t},r=document.documentMode,g={rootMargin:"0px",threshold:0,load:function(t){if("picture"===t.nodeName.toLowerCase()){var e=document.createElement("img");r&&t.getAttribute("data-iesrc")&&(e.src=t.getAttribute("data-iesrc")),t.getAttribute("data-alt")&&(e.alt=t.getAttribute("data-alt")),t.appendChild(e)}t.getAttribute("data-src")&&(t.src=t.getAttribute("data-src")),t.getAttribute("data-srcset")&&(t.srcset=t.getAttribute("data-srcset")),t.getAttribute("data-background-image")&&(t.style.backgroundImage="url('"+t.getAttribute("data-background-image")+"')")},loaded:function(){}};
+/**
+ * Detect IE browser
+ * @const {boolean}
+ * @private
+ */function l(t){t.setAttribute("data-loaded",!0)}var f=function(t){return"true"===t.getAttribute("data-loaded")};return function(){var r,o,a=0<arguments.length&&void 0!==arguments[0]?arguments[0]:".lozad",t=1<arguments.length&&void 0!==arguments[1]?arguments[1]:{},e=s({},g,t),n=e.rootMargin,i=e.threshold,d=e.load,u=e.loaded,c=void 0;return window.IntersectionObserver&&(c=new IntersectionObserver((r=d,o=u,function(t,e){t.forEach(function(t){0<t.intersectionRatio&&(e.unobserve(t.target),f(t.target)||(r(t.target),l(t.target),o(t.target)))})}),{rootMargin:n,threshold:i})),{observe:function(){for(var t,e=(t=a)instanceof Element?[t]:t instanceof NodeList?t:document.querySelectorAll(t),r=0;r<e.length;r++)f(e[r])||(c?c.observe(e[r]):(d(e[r]),l(e[r]),u(e[r])))},triggerLoad:function(t){f(t)||(d(t),l(t),u(t))}}}});
```

Comment:
can you check why this function has been defined separately unlike before?

Reply:
better discard your `.min.js` changes, it'll anyway get overriden by Travis' build

Reply:
Thanks for reviewing, I rolled back changes in dist/ folder.